mlflow/tracing/types/constant.py

```diff
@@ -1,11 +1,14 @@
 # NB: These keys are placeholders and subject to change
 class TraceMetadataKey:
-    NAME = "name"
     INPUTS = "inputs"
     OUTPUTS = "outputs"
     SOURCE = "source"
```

Comment:
Can we rename this to `mlflow.source.name`? we also need `mlflow.source.type`.

Reply:
Makes sense!